modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager, you can grant access to applications that want to consume your APIs by creating instance Contracts. +
+After you grant access to applications in Exchange, a contract is automatically created for the API instance. You can then manage these contracts from API Manager. +
```

Comment:
"After you grant access to applications in Exchange" is a bit confusing. The API Owner may grant access to a client application request in the API in *API Manager*. The client application owner must request access to the API in the Exchange asset corresponding to the API, in *Exchange*.

Reply:
Thanks, Nahuel! I think that's too much detail that is not required here. I simplified it a bit more.